### DISCUSSION TITLE
Micropython binding

### DISCUSSION THREAD

**Discussion by jej** [Bodmer/TFT_eSPI#926] [Compatible support libraries]
Hi,
Do you think there is any chance to bind TFT_eSPI library with Micropython, as an external module? (https://docs.micropython.org/en/latest/develop/cmodules.html). Or it's too much hardware dependent and the bind will conflict with Micropython internals?
Thx for any advice.

**Comment by Bodmer** [comment]
I have no knowledge of python or the environment so am unable to guage the effort involved. [Maybe you can use ArduPy](https://github.com/Seeed-Studio/ArduPy)?

**Comment by jej** [comment]
I didn't know this ArduPy... It looks very Seeed-Studio hardware dependent. I continue the investigation. Thanks for your help, have a good continuation.
